# for flight task
uint8 task  # 1 takeoff, 2 gohome, 3 hover, 4 ctrl

# for horizontal control
uint8 mode  # 1 for pos ctrl, 2 for vel ctrl
float64 x
float64 y

# altitude cmd
float64 z

# global yaw cmd
float64 yaw



